program.py

```diff
@@ -25,7 +24,7 @@
 
 # List of possible endpoint types
 class EndpointTypes(enum.Enum):
-    OCS = 'OCS'
+    ADH = 'ADH'
```

Comment:
same comment about trying to alias both strings to the new adh enum instead of the breaking change of deprecating the OCS value

Reply:
That's a good point, what do you think about this latest change; keeping OCS around along with ADH and add a deprecation note? That shouldn't break current code, and is easy to rip out once we don't support it anymore (I just threw in some 'or' statements, I didn't feel making helper methods for this is necessary since we'll remove it anyway)

Reply:
Good point, I went ahead and added ADH as a new endpoint type, preserving OCS with a deprecetation note. This way any current code should still be good and it will be easy to remove OCS when the time comes (I just threw in some 'or' clauses, nothing fancy since it will be delete). What you do you think about that?

Reply:
I wonder if we can do something a bit more aggressive (but not breaking). So for example, when we read the config file in line 422, we could have an extra couple lines that detect the presence of 'OCS' and change it to 'ADH' in the endpoint["EndpointType"] entry. Then the rest of the sample can just migrate to ADH entirely, and we intercept any instances of OCS right at the point where we read them in (that check of if == 'OCS' could also include a print warning saying to stop using 'OCS' and switch to 'ADH').

Reply:
Changing on the read path now instead, I left the Note comment and added a print statement saying that we are changing their type to OCS in such case.